framework/source/class/qx/io/remote/transport/XmlHttp.js

```diff
@@ -308,6 +332,12 @@ qx.Class.define("qx.io.remote.transport.XmlHttp",
         return;
       }
 
+      // Apply timeout
+      var timeout = qx.io.remote.transport.XmlHttp.getTimeout();
```

Comment:
No, because it's static and being called from a member; AFAICR there is 'self' but personally I find that obscure and not distinct so I prefer the longer form